nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -349,6 +380,82 @@ in {
         Windows can unseal the encryption key.
       '';
     };
+
+    windows = mkOption {
+      default = { };
+      description = ''
+        Make Windows bootable from systemd-boot. This option is not necessary when Windows and
+        NixOS use the same EFI System Partition (ESP). In that case, Windows will automatically be
+        detected by systemd-boot.
+
+        However, if Windows is installed on a separate drive or ESP, you can use this option to add
+        a menu entry for each installation manually.
+
+        The attribute name is the title of the menu entry.
```

Comment:
```suggestion
        The attribute name is the filename of the boot entry.
```

Reply:
Yes, that was still outdated. Should be clearer now.